components/feature/awesomebar/src/main/java/mozilla/components/feature/awesomebar/provider/HistoryMetadataSuggestionProvider.kt

```diff
@@ -70,6 +71,7 @@ class HistoryMetadataSuggestionProvider(
                 editSuggestion = result.key.url,
                 onSuggestionClicked = {
                     loadUrlUseCase.invoke(result.key.url)
+                    emitHistorySuggestionClickedFact()
```

Comment:
Under the assumption that this provider will replace the other one I think this is fine and we can compare events between versions. This is what we currently have but I have a feeling that this will have to change and we will have both providers active. We only keep metadata for a limited amount of time and I think we will want the original history provider as well. However, this currently matches the ticket. Let's discuss and iterate in the coming weeks.